rules-tests/CodeQuality/Rector/MethodCall/AssertCompareToSpecificMethodRector/Fixture/count.php.inc

```diff
@@ -25,6 +27,8 @@ final class Count extends \PHPUnit\Framework\TestCase
     public function test()
     {
         $this->assertCount(5, $something);
+        $this->assertSame(count($something), 5);
```

Comment:
Could you extract this to a new test fixture? It looks unrelated to change here

Reply:
Done